packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -48,7 +48,7 @@ class FuchsiaSdk {
   /// Example output:
   ///    $ device-finder list -full
   ///    > 192.168.42.56 paper-pulp-bush-angel
-  Future<String> listDevices({ Duration timeout }) async {
+  Future<List<String>> listDevices({ Duration timeout }) async {
```

Comment:
This change will require an update in google3.

Reply:
Crap :(

Reply:
Updated API